wagtail/test/settings.py

```diff
@@ -171,6 +171,7 @@
     "wagtail",
     "taggit",
     "rest_framework",
+    "django_filters",
```

Comment:
This is so that the `django_filters/widgets/multiwidget.html` template is picked up by the template loader. I wonder if we should've included this in the project template as well? [According to the docs](https://django-filter.readthedocs.io/en/stable/guide/install.html), you are supposed to add it to the `INSTALLED_APPS`.